parlai/crowdsourcing/utils/tests.py

```diff
@@ -408,3 +408,26 @@ def _send_agent_message(
             "episode_done": False,
         }
         self.server.send_agent_act(agent_id=agent_id, act_content=act_content)
+
+
+def check_stdout(actual_stdout: str, expected_stdout_path: str):
```

Comment:
I would strongly advise we use a pytest regression

Reply:
Yes, normally I would do a pytest regression for this. However, this is a special case, because we want to check the stdout against some reference stdout, but the actual stdout will often but not always include strings representing progress indicators - for instance, see the "`40%|####      | 4/10 [00:00<00:00, 15.52it/s]`" string at https://app.circleci.com/pipelines/github/facebookresearch/ParlAI/8467/workflows/5621488b-c27a-46e8-a76e-3993dee8dd59/jobs/68923 . Thus, this PR checks the actual stdout against the reference stdout in a way that is tolerant to these extra strings, but as a result doesn't use pytest regressions